taxi/models.py

```diff
@@ -15,7 +16,11 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[RegexValidator(regex=r"^[A-Z]{3}\d{5}$")]
```

Comment:
The `license_number` field in the `Driver` model uses a `RegexValidator` to enforce a specific format. Ensure that this format (three uppercase letters followed by five digits) matches the business requirements for license numbers.